services/service-core/src/auth/keycloak.service.ts

```diff
@@ -54,13 +61,13 @@ export class KeycloakService {
       );
 
       if (!getTokenData.ok) {
-        throw new Error('Error: Fetch error');
+        throw new Error(getTokenData.statusText);
```

Comment:
This and all other error messages in this PR are very non-descriptive. We have just implemented another PR where we did just that - fixed non-descriptive error messages and made them better. Here we introduce the same problem again. All error message need to be reviewed and improved